configuration/pih/htmlforms/hiv/section-hiv-history.xml

```diff
@@ -1523,19 +1525,18 @@
                             <tr class="startDateEndDate">
                                 <td>
                                     <obs conceptId="CIEL:1282" answerConceptId="{conceptH}"
-                                         answerLabel="{drugname}"
-                                         toggle="{id: '{drugname}-hx', style: 'dim'}" />
-                                    <p class="radio {drugname}-hx">
+                                         answerLabel="{drugname}" toggle="{id: '{drugname}-hx', style: 'dim'}" id="{drugname}"/>
```

Comment:
If we are writing our own custom function instead of using the existing "toggle" functionality do we want to remove the toggle here?

Reply:
the toggle function here works on rows and dates at the same time. In my case, when we select a row, I activate the date fields.